Cargo.toml

```diff
@@ -115,14 +117,20 @@ wayland-backend = { version = "0.1.0", default_features = false, features = ["cl
 wayland-protocols = { version = "0.30.0", features = [ "staging"], optional = true }
 calloop = "0.10.5"
 x11-dl = { version = "2.18.5", optional = true }
+drm = { version = "0.6.2", optional = true }
+input = { version = "0.7.1", optional = true }
+libseat = { version = "0.1.4", optional = true }
+udev = { version = "0.6.3", optional = true }
+xkbcommon = { version = "0.5.0", optional = true }
```

Comment:
It might be worth it to just use `xkbcommon-dl` instead, since that's what #2789 is using.